api/src/main/webapp/static/README.md

```diff
@@ -0,0 +1,8 @@
+#Static Files in AppEngine
+We host a number of public static files in GAE for cluster creation. These files are copied at build time from our cluster resources directory. Eventually (when we no longer support bucket extensions), these files should move into this directory and we should no longer gitignore them from here.
+
```

Comment:
Since we're globbig the directory, won't this be served as a static resource? If so, I might suggest moving this to either api/doc or cluster-resources/README.md

Reply:
This will be served as a static resource, but I think that's probably okay? I'd prefer to have it where the files are. Unless you have strong feelings against it being served? (I can also add it as a specific exclude?)

Reply:
Good idea - exclude sounds reasonable